docs/unity/part-1.md

```diff
@@ -12,14 +12,19 @@ This tutorial has been tested against UnityEngine version 2022.3.4f1. This tutor
 
 ## Prepare Project Structure
 
-This project is separated into two sub-projects, one for the server (module) code and one for the client code. First we'll create the main directory, this directory name doesn't matter but we'll give you an example:
+This project is separated into two sub-projects;
+
+1. Server (module) code 
+2. Client code.
+
+First, we'll create an arbitrarily-named project root directory:
```

Comment:
What do you think about this?
> First, we'll create a project root directory (you can choose the name):

For some reason, the word "arbitrary" in this context makes me think that the naming shouldn't matter to _me_, either.

Reply:
Takes focus off the name - better (changing to this)